### PR TITLE
Update lisk faucet to support lisk sdk v4.0.0

### PR DESCRIPTION
**Pull request by MichalTuleja**
- Update Lisk Elements to version 4.0.0

### CODE REVIEW

**Review by shuse2 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lisk-faucet",
-  "version": "2.1.0-rc.0",
+  "version": "3.0.0-beta.0",
```
Comment (shuse2) [inline package.json:3]:
```suggestion
  "version": "3.0.0-beta.1",
```

File: api/lisk.js [hunk 2]
```diff
@@ -169,18 +166,39 @@ module.exports = function (app) {
...
+                        const amount = app.locals.amountToSend * req.fixedPoint;
+                        const networkIdentifier = cryptography.getNetworkIdentifier(app.locals.nethash, 'Lisk');
+                        const transaction = transactions.transfer(
+                            {
+                                networkIdentifier,
+                                recipientId: address,
+                                amount: String(amount),
+                                passphrase: app.locals.passphrase,
+                                nonce: account.nonce,
+                                fee: MINIMUM_FEE,
```
Comment (MichalTuleja) [inline api/lisk.js:191]: So fee is always the minimum? Does it make sense to involve the estimates from Lisk Service here?

Comment (shuse2) [reply api/lisk.js:191]: i think for now, minimum_fee should be good for now, but maybe we should consider in the v5

Comment (ManuGowda) [reply api/lisk.js:191]: yea.. for now we can keep minimum fee @MichalTuleja also its only token transfer.

**Pull request merged**